scripts/build.js

```diff
@@ -41,7 +41,9 @@ const main = async () => {
 
   // Copy addresses from @nexusmutual/deployments package
   const addressesFile = path.join(__dirname, '../node_modules/@nexusmutual/deployments/dist/addresses.json');
-  fs.copyFileSync(addressesFile, path.join(dist, 'data/addresses.json'));
+  const dataOutDir = path.join(dist, 'data');
+  fs.mkdirSync(dataOutDir, { recursive: true });
```

Comment:
Since `product-logos.json` is no longer present in the `data` folder, there no longer _is_ a `data` folder, which means we have to create it manually here